tendrl/monitoring_integration/grafana/alert_utils.py

```diff
@@ -14,9 +14,9 @@
 
 def get_alert_dashboard(dashboard_name):
     if dashboard_name == "nodes":
-        slug = "alerts-tendrl-gluster-hosts"
+        slug = "alerts-hosts"
```

Comment:
While creating alert dashbaord we are adding "alerts" as a prefix with dashboard name, so the problem here is dashbaord name is volume-dashbaord then slug name should be like alerts-volume-dashbaord

one more problem is while creating dashbaord name volumes is received from dashbaord_name so the 
dashbaord name is always coming like hosts, volumes or bricks so dashboard name is formed like 
alerts-volumes-dashbaord only. either need to pass dashboard name as volume instead of volumes or need to change in dashbaord name as volumes-dashboard

Reply:
Because using resource name like volumes or bricks or hosts i am identifying dashbaord json tendrl-gluster-volumes.json so if we change resource name as volume then we need to change file name also like tendrl-gluster-volume.json because using resource name only i am forming file name to read dashbaord json.